build.gradle.kts

```diff
@@ -174,8 +171,8 @@ publishing {
         val nexusPassword: String? by project
         if (nexusUsername != null && nexusPassword != null) {
             maven {
-                val releasesRepositoryUrl = "https://mvn.intellectualsites.com/content/repositories/releases/"
-                val snapshotRepositoryUrl = "https://mvn.intellectualsites.com/content/repositories/snapshots/"
+                val releasesRepositoryUrl = "https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/"
```

Comment:
I'm not sure about that at all, but wouldn't ``https://s01.oss.sonatype.org/content/repositories/releases/`` be better

Reply:
If you push a release to the central repository, it's stored in a staging database only you